<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="ViewportFrame">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="L_MainLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="WS::Core::GameDisplay" name="Viewport">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="M_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="A_New"/>
    <addaction name="A_Open"/>
    <addaction name="A_Save"/>
    <addaction name="A_SaveAs"/>
   </widget>
   <widget class="QMenu" name="M_Edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="A_Delete"/>
    <addaction name="A_Copy"/>
    <addaction name="A_Cut"/>
    <addaction name="A_Paste"/>
    <addaction name="separator"/>
    <addaction name="A_Undo"/>
    <addaction name="A_Redo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_WorldStory"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="M_File"/>
   <addaction name="M_Edit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QToolBar" name="T_File">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="A_New"/>
   <addaction name="A_Open"/>
   <addaction name="A_Save"/>
  </widget>
  <widget class="QToolBar" name="T_Edit">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="A_Delete"/>
   <addaction name="A_Copy"/>
   <addaction name="A_Cut"/>
   <addaction name="A_Paste"/>
  </widget>
  <widget class="QToolBar" name="T_UndoRedo">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="A_Undo"/>
   <addaction name="A_Redo"/>
  </widget>
  <widget class="QDockWidget" name="D_Attributes">
   <property name="windowTitle">
    <string>Attributes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="AttributeTable"/>
  </widget>
  <widget class="QDockWidget" name="D_ElementBox">
   <property name="windowTitle">
    <string>Element Box</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="ElementBox"/>
  </widget>
  <action name="A_New">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>Start the creation of a new level.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="A_Open">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Edit the chosen level file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="A_Save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the level currently being edited.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="A_SaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save this level to a different file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="A_Copy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy currently selected level element.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="A_Cut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Move the currently selected level element.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="A_Paste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste the copied level element.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="A_Undo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo last edition.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="A_Redo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo last undo.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="A_Delete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete selected level element</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About WorldStoryCreator...</string>
   </property>
  </action>
  <action name="actionAbout_WorldStory">
   <property name="text">
    <string>About WorldStory...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WS::Core::GameDisplay</class>
   <extends>QGraphicsView</extends>
   <header>classes/gamedisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
